src/main/scala/thurloe/dataaccess/HttpSendGridDAO.scala

```diff
@@ -4,7 +4,7 @@ import akka.http.scaladsl.model.StatusCodes
 import com.sendgrid.SendGrid.Response
 import com.sendgrid._
 import com.typesafe.scalalogging.LazyLogging
-import org.broadinstitute.dsde.rawls.model.{RawlsUserEmail, RawlsUserSubjectId}
```

Comment:
I'd love to get rid of all of the rawls model junk in here, but for now I just cleaned up what was in my way